homeassistant/components/aussie_broadband/__init__.py

```diff
@@ -22,6 +23,17 @@
 PLATFORMS = [Platform.SENSOR]
 
 
+def validate_service_type(service: dict[str, Any]) -> None:
+    """Override for the original class's validation method - Check the service types against known types."""
```

Comment:
Docstrings shouldn't be longer than 72 characters per line. The first sentence is a header and should fit on a single line. After that can come an optional body separated by a blank line.